ne/README.md

```diff
@@ -2,17 +2,17 @@
 
 ![Intro Page](../.gitbook/assets/intro.png)
 
 Programming languages are artificial languages used to give instructions to computers. They are used in most programming tasks and are based on the way humans communicate with each other. Like human languages, programming languages allow words and phrases to be combined to express new concepts. It is interesting to note that the most effective way to communicate with computers involves using a language that is similar to human language.
 
-प्रोग्रामिंग भाषाहरू कम्प्युटरलाई निर्देशन दिन कृत्रिम भाषाहरू हुन्। तिनीहरू प्राय: प्रोग्रामिंग कार्यहरूमा प्रयोग गरिन्छ र व्यक्तिहरूले एक अर्कासँग कुराकानी गर्ने तरीकामा आधारित छन्। मानवीय भाषाहरूले जस्तै प्रोग्रामिंग भाषाहरू नयाँ अवधारणाहरू व्यक्त गर्न शब्दहरू र वाक्यांशहरू प्रदान गर्न अनुमति दिन्छन्। कम्प्युटरसँग कुराकानी गर्ने सबैभन्दा प्रभावकारी तरिका भनेको मानव भाषासँग मिल्दोजुल्दो भाषा प्रयोग गर्नु हो र यो निकै नै चाखलाग्दो विषय छ।
+प्रोग्रामलाई एउटा सफ्टवेयरले कम्प्युटरसँग अन्तर्क्रिया गर्ने तरिकाको रूपमा पनि लिन सकिन्छ। जसरी मानिसहरूसँग सञ्चार गर्न धेरै भाषाहरू छन्, कम्प्युटरसँग पनि आफ्नै सञ्चारको लागि विभिन्न भाषाहरू छन्। यस कृत्रिम भाषा जुन कम्प्युटरहरूलाई निर्देशन दिन प्रयोग गरिन्छ त्यसलाई  Programming Language भनिन्छ। उदाहरण को लागी BASIC, FORTRAN, Javascript, Python, C, C++, Dart आदि। यी programming language धेरै लामो समयदेखि विकसित हुँदै आएको छ जसले कम्प्यूटर लगाएत मानिसहरूलाई  पनि त्यो विशेष भाषा बुझ्न सजिलो बनाइरहेको छ। अहिले त पुराण भाषालाई निकै नै नयाँले प्रतिस्थापन गरिदिएको छ, जुन सिक्न सजिलो हुनुको साथ साथै विभिन्न क्षेत्रमा उपयोग गर्न सकिन्छ। यी सबै पुराना भाषाहरू मध्ये, जाभास्क्रिप्ट(Javascript) एक पुरानो तर प्रभावशाली प्रोग्रामिङ भाषा हो जुन टेक्नोलोजीको लगभग हरेक भागमा प्रयोग गरिन्छ।
```

Comment:
Didn't get this part. 
`अहिले त पुराण भाषालाई निकै नै नयाँले प्रतिस्थापन गरिदिएको छ`

You mean to say? 
`अहिले त पुराना भाषालाई निकै नै नयाँले प्रतिस्थापन गरिदिएको छ`

Reply:
oh exactly. I typed them all by myself and some are being typos and some I don't know maybe I don't know the spellings well hehe.

Reply:
No worries, it happens. It's nice to have genuine content.